libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/internal/event/AppUserTurnstileInterface.kt

```diff
@@ -0,0 +1,7 @@
+package com.mapbox.navigation.metrics.internal.event
+
+import com.mapbox.common.TurnstileEvent
+
+interface AppUserTurnstileInterface {
```

Comment:
Do you need this interface? You implement it but never use it.

Reply:
not used anymore, removed